configurator/TEST-org.osgi.test.cases.configurator-7.0.0.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Isn't this file normally what is produced during a surefire test run. Do we need it as part of the sources?

Reply:
No, you need to run the OSGi test suite and that is restricted and not part of the build sadly. I created a bnd workspace and created a run configuration to run these test suites. I am not sure how this is normally handled but thought it best to include it as proof that it passes the OSGi CT.